app/serializers/degree_serializer/v01.rb

```diff
@@ -2,7 +2,7 @@
 
 module DegreeSerializer
   class V01
-    EXCLUDE_ATTRIBUTES = %w[
+    EXCLUDED_ATTRIBUTES = %w[
```

Comment:
So, having talked to Adam, we think that it makes sense to hide `locale_code` from end users, because only seems to be used internally.